torch/_dynamo/eval_frame.py

```diff
@@ -861,6 +861,9 @@ def argument_names(f: Callable[..., Any], *args, **kwargs) -> List[str]:
 
     new_graph.recompile()
 
+    # TODO remove this once Executorch uses proper functionalization
+    new_graph._example_fake_inputs = example_fake_inputs
```

Comment:
I have no context for why we need this, or why we are leaking fake tensors out, but I also don't wanna be difficult.